tools/network-scene-vis.md

```diff
@@ -0,0 +1,50 @@
+---
+id: netscenevis
+title: Network Scene Visualization
+---
+
+The Network Scene Visualization (NetSceneVis) is a powerful, user-friendly tool included in the Multiplayer Tools package to help you visualize and debug network communication on a per-object basis in the scene view of your project with visualizations such as mesh shading and text overlay.
```

Comment:
One other question (just to confirm), in Unity documentation do we usually refer to users in second person ("you")?

Reply:
Yes, it is.